### DISCUSSION TITLE
3 Important Features

### DISCUSSION THREAD

**Discussion by ArminEskandarii** [Floorp-Projects/Floorp#1234] [Feature Request]
Hi, First of all the project is amazing you guys are awesome.
Feature 1: I wanna be able to group tabs (This feature is available in Chrome)
2 Feature 2: I wanna be able to remove this gap from the top and the tab page.
![image](https://github.com/Floorp-Projects/Floorp/assets/106415257/ef7407af-0058-4aec-b75a-4b71d16844e4)
Feature 3: I wanna be able to remove these icons.
![Screenshot (30)](https://github.com/Floorp-Projects/Floorp/assets/106415257/59d2d469-68aa-4ced-83a7-a7e4fc40c0c3)

**Comment by CutterKnife** [comment]
The tab group feature (like Chrome) was abandoned due to technical difficulties.

**Comment by breadbeforerice** [comment]
2, how about putting this in userChrome.css for example
#urlbar-container {
margin-top: -1.7px !important;
}
3, there is an option to disable the Lepton context menu and panel menu icons in the Design -> Lepton settings in the preferences.

**Reply by ArminEskandarii** [reply]
I'm using Floorp Fluerial UI  not lepton.
That user Chrome thing didn't work, Can you help me with it please.

**Reply by breadbeforerice** [reply]
I was wrong about the location. Try this.
#TabsToolbar-customization-target {
	margin-top: -15px;
}

**Reply by ArminEskandarii** [reply]
> I was wrong about the location. Try this. #TabsToolbar-customization-target { margin-top: -15px; }

Bro it worked man, IDK how to thank you 😍😍

**Reply by breadbeforerice** [reply]
We hope you don't mind.
3 is now ready, if you'd like to try it.
#appMenu-fxa-text{
margin-left:-6px!important;
}
#appMenu-zoom-controls{
margin-left:-40px!important;
}
#appMenu-zoomReduce-button2{
margin-right: -15px!important;
}
#appMenu-new-tab-button2,#appMenu-new-window-button2,#appMenu-new-private-window-button2,
#appMenu-ssb-button,#appMenu-bookmarks-button,#appMenu-history-button,#appMenu-downloads-button,
#appMenu-passwords-button,#appMenu-extensions-themes-button,#openprofiledir,
#appMenu-print-button2,#appMenu-save-file-button2,#appMenu-find-button2,#appMenu-translate-button,
#appMenu-settings-button,#appMenu-more-button2,#appMenu-help-button2,#rebootappmenu,#appMenu-quit-button2{
margin-left:-31px!important;
}

**Reply by ArminEskandarii** [reply]
> We hope you don't mind. 3 is now ready, if you'd like to try it. #appMenu-fxa-text{ margin-left:-6px!important; } #appMenu-zoom-controls{ margin-left:-40px!important; } #appMenu-zoomReduce-button2{ margin-right: -15px!important; } #appMenu-new-tab-button2,#appMenu-new-window-button2,#appMenu-new-private-window-button2, #appMenu-ssb-button,#appMenu-bookmarks-button,#appMenu-history-button,#appMenu-downloads-button, #appMenu-passwords-button,#appMenu-extensions-themes-button,#openprofiledir, #appMenu-print-button2,#appMenu-save-file-button2,#appMenu-find-button2,#appMenu-translate-button, #appMenu-settings-button,#appMenu-more-button2,#appMenu-help-button2,#rebootappmenu,#appMenu-quit-button2{ margin-left:-31px!important; }

Bro I don't know how to thank you man, I just wanna say what a supportive community
